callback_plugins/aa_version_requirement.py

```diff
@@ -31,6 +31,10 @@ def display(*args, **kwargs):
 # Set to minimum required Ansible version
 REQUIRED_VERSION = '2.2.0.0'
 DESCRIPTION = "Supported versions: %s or newer" % REQUIRED_VERSION
+FAIL_ON_2_2_1_0 = "There are known issues with Ansible version 2.2.1.0 which " \
+                  "are impacting OpenShift-Ansible. Please downgrade to " \
+                  "Ansible version 2.2.0.0. See this issue for more details: " \
```

Comment:
I can update the guidance.  And it would probably be good to update the `or newer` reference so someone doesn't upgrade from an older version only to find out they have to downgrade if they went to 2.2.1.0.